src/coreclr/jit/codegenarm64.cpp

```diff
@@ -8395,6 +8395,9 @@ void CodeGen::genArm64EmitterUnitTests()
     theEmitter->emitIns_R_R_R(INS_cmeq, EA_8BYTE, REG_V12, REG_V13, REG_V14, INS_OPTS_2S);
     theEmitter->emitIns_R_R_R(INS_cmeq, EA_16BYTE, REG_V15, REG_V16, REG_V17, INS_OPTS_4S);
     theEmitter->emitIns_R_R_R(INS_cmeq, EA_16BYTE, REG_V18, REG_V19, REG_V20, INS_OPTS_2D);
+    theEmitter->emitIns_R_R(INS_cmeq, EA_8BYTE, REG_V0, REG_V1);
+    theEmitter->emitIns_R_R(INS_fcmeq, EA_8BYTE, REG_V2, REG_V3);
```

Comment:
The `fcmeq` tests should be for `4_BYTE` (`float`) and `8_BYTE`(`double`).

Can you also move the last two tests down (between `// faddp vector` and `// fcmeq vector`) and add comment `// fcmeq scalar`.

The `cmeq` test is also needs to be moved to a separate group with `// cmeq scalar` comment.

Reply:
Done, also I think we already have tests for fcmeq:
```cpp
    // fcmeq Vd, Vn, #0.0
    theEmitter->emitIns_R_R(INS_fcmeq, EA_4BYTE, REG_V0, REG_V1); // scalar 4BYTE
    theEmitter->emitIns_R_R(INS_fcmeq, EA_8BYTE, REG_V2, REG_V3); // scalar 8BYTE
```